e2e/tests/omnibus-sepolia.toml/BFP_Market_Liquidations.e2e.js

```diff
@@ -0,0 +1,665 @@
+      args: [marketId],
+    });
+
+    const updatedMarketConfiguration = {
+      ...marketConfiguration,
+      minMarginUsd: oldMinMarginUsd,
+    };
+
+    await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'setMarketConfigurationById',
+      args: [
+        {
+          marketId,
+          ...updatedMarketConfiguration,
+        },
+      ],
+      impersonate: owner,
+    });
+  });
+
+  it('should open a short with sUSD collateral', async () => {
+    const collateralAddress = require('../../deployments/systemToken.json').address;
+
+    await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'modifyCollateral',
+      args: [accountId, marketId, collateralAddress, ethers.utils.parseEther(String(20_000))],
+    });
+
+    const newDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getAccountDigest',
+      args: [accountId, marketId],
+    });
+    log({ newDigest });
+
+    const newDepositedsUSD = newDigest.depositedCollaterals.find(
+      (c) => c.collateralAddress === collateralAddress
+    );
+    log({ newDepositedsUSD });
+    const { now, blockTimestamp } = getTimes(provider);
+    const buffer = 1; // 1s
+    const diff = now - blockTimestamp;
+
+    if (diff < 0) {
+      await wait(Math.abs(diff) + buffer * 1000);
+    }
+    await commitBfpOrder({
+      wallet,
+      accountId,
+      marketId,
+      sizeDelta: -0.01,
+    });
+    await wait(1000);
+    await wait(5000);
+
+    const newPosition = await settleBfpOrder({ wallet, accountId, marketId });
+    assert.equal(newPosition.positionSize, -0.01);
+  });
+
+  it('should update market configuration minMarginUsd force liquidations', async () => {
+    let marketConfiguration = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getMarketConfigurationById',
+      args: [marketId],
+    });
+    oldMinMarginUsd = marketConfiguration.minMarginUsd;
+    log({ marketConfiguration });
+
+    const marginDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getMarginDigest',
+      args: [accountId, marketId],
+    });
+    log({ marginDigest });
+    const collateralUsd = marginDigest.collateralUsd;
+    log({ collateralUsd });
+
+    // Sets minMarginUsd to be above the current collateral value
+    const updatedMarketConfiguration = {
+      ...marketConfiguration,
+      minMarginUsd: collateralUsd + 1,
+    };
+
+    const owner = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'owner',
+    });
+
+    await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'setMarketConfigurationById',
+      args: [
+        {
+          marketId,
+          ...updatedMarketConfiguration,
+        },
+      ],
+      impersonate: owner,
+    });
+  });
+
+  it('should flag an underwater sUSD backed position', async () => {
```

Comment:
Do we need to have sUSD liquidation test together with ETH liquidation test? From what I can see it is a 1-1 copy for ETH. So unless there is a check here how these two separate positions affect each other - we should separate sUSD liquidation into its own file and reduce this one in half.

Reply:
ok so I compared these two liquidations and they are absolutely identical. Please make a separate file for sUSD liquidation


![Screenshot 2024-09-20 at 18 54 37](https://github.com/user-attachments/assets/d7699ddf-968c-4b6f-a84e-01830fff2c08)